### PR TITLE
github icon features added

### PR DESCRIPTION
**Pull request by Varun8216889**
### Pull Request

This PR fixes 

1. What type of change does your PR bring? 
    - [ ] Bug fix
    - [ ] Feature Update
    - [x] Issue fix
    - [ ] Doc fix/update
    - [ ] Other

2. Any else you want to convey? (Additional context)

### CODE REVIEW

**Comment by alphaX86** [comment]
Um, you could have unchanged that work flow file

**Comment by alphaX86** [comment]
Maybe I'll open a separate issue for it. Then you can work on

**Comment by Varun8216889** [comment]
ok
 I linked the icon on the about page but there is still an error about 'All jobs have failed'

**Comment by alphaX86** [comment]
Don't need to worry on that, we're trying to figure out

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -5,11 +5,11 @@
 # file with configuration.  For more information, see:
 # https://github.com/actions/labeler
 
-name: Labeler
+name: 
 on: [pull_request]
 
 jobs:
-  label:
+  label:level0
```
Comment (alphaX86) [inline .github/workflows/label.yml]: Revert those changes alone

**Review by alphaX86 — CHANGES_REQUESTED** [review]

File: .github/workflows/label.yml (hunk 1 above)
Comment (alphaX86) [reply .github/workflows/label.yml]: Bring them back

**Review by alphaX86 — APPROVED** [review]
LGTM 👍

**Comment by Varun8216889** [comment]
thank u so much for showing faith in me and for giving me a chance to contribute.
it is my first PR and I am so happy. It is all because of u.

**Comment by alphaX86** [comment]
@Varun8216889 You're welcome and keep contributing bud! You got this! 👍

**Comment by Varun8216889** [comment]
if u find some issue plz let me know. I would love to contribute

**Pull request merged**